app.js

```diff
@@ -5,7 +5,10 @@ var eyros = require('eyros/2d')
 var mixmap = require('mixmap')
 var resl = require('resl')
 var regl = require('regl')
-var httpStorage = require('mixmap-peermaps/storage/http')
+
+var createStorage = require('./storage')
+var createHttpBackend = require('./storage/http')
+var createHyperdriveBackend = require('./storage/hyperdrive')
 
```

Comment:
I put these abstractions locally for now. It makes development a bit quicker. We can move this down to `mixmap-peermaps` once we're happy with it.